reframe/frontend/cli.py

```diff
@@ -77,6 +77,20 @@ def fmt_list(x):
 
         lines.append('')
 
+    lines.append('    Dependencies for current system:')
```

Comment:
Couldn't you add this as an element in `check_info`? And similarly to `fmt_list()`, have something like `fmt_deps()`.